rust/src/operations/transaction/mod.rs

```diff
@@ -164,7 +164,8 @@ pub async fn commit(
 ) -> DeltaResult<i64> {
     let tmp_commit = prepare_commit(storage, &operation, actions, app_metadata).await?;
 
-    let max_attempts = 5;
+    // TODO make max attempts configurable
+    let max_attempts = 25;
```

Comment:
Increasing this was needed to make our concurrent writer tests pass.

Reply:
Huh. I guess 5 workers trying to make 3 commits. That mean one potentially has to fail 14 times?

Reply:
10 was not cutting it for s3, so at least that...